cloud-controller-manager/do/loadbalancers.go

```diff
@@ -403,6 +442,7 @@ func (l *loadBalancers) EnsureLoadBalancerDeleted(ctx context.Context, clusterNa
 		return err
 	}
 
+	klog.V(2).Infof("deleting do loadbalancer: %s", lb.ID)
```

Comment:
Log messages should be capitalized and wording be made consistent with the pre-existing style, i.e., `Deleting load-balancer %s`.